### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by thecoya0327** [aclap-dev/video-downloadhelper#1171] [License & Purchase requests]
### What's your issue?

What's your issue?
I bought it wrong for Firefox I'd like a refund
I'd like to request a refund.

### The 10 first digits (only 10) of your license number or email address

b0ca2bf9e9

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
